docs/proposals/images/machine-states-preboot/Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SOURCES := $(shell ls *.plantuml)
+FIGURES := $(SOURCES:%.plantuml=%.png)
+
+figures: $(FIGURES)
+
+# --user ensures the output files are owned by the user invoking docker on the host
+%.png: %.plantuml
+	docker run \
+		--rm \
+		--volume ${PWD}:/figures \
+		--user $(shell id --user):$(shell id --group) \
+		dpf9/plantuml:1.2019.6 \
```

Comment:
I have no objections to using this image as a stopgap, but we should create an issue to track getting this image built and published through the k8s infra work.

Reply:
I'll file an issue

Reply:
When you have the issue filed, please add a TODO comment in the code, then lgtm